build.sbt

```diff
@@ -1,6 +1,6 @@
 organization  := "com.gu"
 description   := "AWS Lambda to send email to stakeholders for expired sponsorships."
-scalaVersion  := "2.12.4"
+scalaVersion  := "2.12.19"
```

Comment:
Without upgrading to this more recent version of Scala 2.12, we would get [a very long `StackOverflowError`](https://gist.github.com/rtyley/2db314d5015f4f3a5f1b1d26ed53c59e) trying to compile this project.

Ideally, we would want to upgrade to Scala 2.13 or Scala 3, but this smaller jump requires a bit less work for now, and resolves the compiler problem.